python-template/{{cookiecutter.placeholder_repo_name}}/requirements/pip-tools.txt

```diff
@@ -5,8 +5,8 @@
 #    make upgrade
 #
 click==7.1.2              # via pip-tools
-pip-tools==5.1.2          # via -r requirements/pip-tools.in
-six==1.14.0               # via pip-tools
+pip-tools==5.4.0          # via -r python-template/{{cookiecutter.placeholder_repo_name}}/requirements/pip-tools.in
+six==1.15.0               # via pip-tools
 
 # The following packages are considered to be unsafe in a requirements file:
-# pip
+pip==20.3.1               # via pip-tools
```

Comment:
You need to re-run the command after your edit for this to get commented out again.